seed_data/challenges/computer-science.json

```diff
@@ -44,8 +49,12 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Waypoint: Aprende Loops",
+      "descriptionEs": [
+        "Ahora apuntaremos a completar la Semana 2 del curso de Introducción a Informática de Stanford.",
+        "Esto nos introducirá a los loops, una característica fundamental de cada lenguaje de programación",
```

Comment:
I would add the word "bucle" as translation for loop, just as a reference for Spanish speakers who have seen the term before.
So the paragraph would be: "Esto nos introducirá a los loops *(bucles)*, una característica fundamental de *todos los* lenguajes de programación."